Realm/ObjectServerTests/RLMPermissionsAPITests.m

```diff
@@ -19,7 +19,6 @@
 #import <XCTest/XCTest.h>
 
 // FIXME: Many permission tests appears to fail with the ROS 3.0.0 alpha releases.
```

Comment:
Is this still true? If not, we should remove the FIXME as well.

Reply:
Yes, most of the tests for the old permissions API fail. The files are now not compiled at all.